src/Symfony/Component/FeatureFlag/composer.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "symfony/feature-flag",
+    "type": "library",
+    "description": "Provide a feature flag mechanism.",
```

```suggestion
    "description": "Provides a feature flag mechanism",
```

It seems the form "3rd person / no final perdio"  is the most employed in the composer descriptions 🤷